src/main/java/core/basesyntax/ValidatorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class ValidatorImpl implements Validator {
+    public static final String PATTERN = "[bspr],(banana|apple),\\d+";
```

Comment:
```suggestion
    private static final String PATTERN = "[bspr],(banana|apple),\\d+";
```
так як тільки цьому класі нам потрібен краще його приватизувати